src/test/java/io/xstefank/wildfly/bot/PRCommitsQuantityCheckTest.java

```diff
@@ -46,26 +50,34 @@ void configFileNullTest() {
 
     @Test
     void tooManyCommitsCheckFailTest() throws IOException {
+        jsonHandlerBuilder.setCommitQuantity(3);
+        jsonHandlerBuilder.setAction(Action.EDITED);
+        JsonHandler jsonHandler = jsonHandlerBuilder.build();
```

Comment:
I think this should be rather written as following for clearer code:

`JsonHandler jsonHandler = new JsonHandler.JsonHandlerBuilder("pr-template.json")
                                                  .setCommitQuantity(3)
                                                  .setAction(Action.EDITED)
                                                  .build();`

Reply:
new JsonHandler.JsonHandlerBuilder() ?? :)